### PR TITLE
Commerce 4843 added missing label

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/health/status/CommerceGuestCheckoutAuthenticationCommerceHealthStatus.java [hunk 1]
```diff
@@ -144,7 +144,7 @@ public void fixIssue(long companyId, long commerceChannelId)
 
 			List<JournalArticle> journalArticles =
 				_journalArticleLocalService.getArticlesByStructureId(
-					commerceChannel.getSiteGroupId(),
+					serviceContext.getScopeGroupId(),
```
Comment (dsitu) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/health/status/CommerceGuestCheckoutAuthenticationCommerceHealthStatus.java:147]: @andrea-ale-sbarra I think this should be in a separate commit, not part of the build lang.

**Review by dsitu — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/health/status/CommerceGuestCheckoutAuthenticationCommerceHealthStatus.java (hunk 1 above)
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/health/status/CommerceGuestCheckoutAuthenticationCommerceHealthStatus.java]: Yes, thank you @dsitu I did a mistake by adding that class in this commit, I've removed it.

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by marco-leo** [comment]
ci:forward
